packages/ruby/lib/readme/har/response_serializer.rb

```diff
@@ -21,7 +21,7 @@ def as_json
           content: content,
           redirectURL: @response.location.to_s,
           headersSize: -1,
-          bodySize: @response.content_length,
+          bodySize: @response.content_length || 0,
```

Comment:
Would it make sense to get the size of the raw body here instead? This is how the middleware does it:

```ruby
  if !STATUS_WITH_NO_ENTITY_BODY.include?(status.to_i) &&
     !headers['Content-Length'] &&
     !headers['Transfer-Encoding'] &&
     !(@sendfile && headers[@sendfile])

    old_body = body
    body, length = [], 0
    old_body.each do |part|
      body << part
      length += bytesize(part)
    end
    old_body.close if old_body.respond_to?(:close)
    headers['Content-Length'] = length.to_s
  end
```

If we decide to go down that path, we might want to move this into the `HttpRequest` object